net.codeindustry.MasterPDFEditor4.yaml

```
@@ -0,0 +1,46 @@
+app-id: net.codeindustry.MasterPDFEditor4
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: masterpdfeditor4
+finish-args:
+  - '--device=dri'
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=wayland'
+modules:
+  - name: sane-backends
+    sources:
+      - type: archive
+        url: https://gitlab.com/sane-project/backends/uploads/9e718daff347826f4cfe21126c8d5091/sane-backends-1.0.28.tar.gz
+        sha256: 31260f3f72d82ac1661c62c5a4468410b89fb2b4a811dabbfcc0350c1346de03
+    post-install:
+      - 'echo 127.0.0.1 > /app/etc/sane.d/net.conf'
+      - 'echo net > /app/etc/sane.d/dll.conf'
+
+  - name: masterpdfeditor4
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p /app/bin'
+      - 'ln -s /app/{extra,bin}/masterpdfeditor4'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 644 net.codeindustry.MasterPDFEditor4.metainfo.xml -t /app/share/metainfo'
+      - 'install -Dm 644 net.codeindustry.MasterPDFEditor4.desktop -t /app/share/applications'
+      - 'install -Dm 644 net.codeindustry.MasterPDFEditor4.png -t /app/share/icons/hicolor/128x128/apps'
+    sources:
+      - type: extra-data
+        url: http://www.code-industry.net/public/
+        filename: master-pdf-editor-4.3.89_qt5.amd64.tar.gz
```

```suggestion
        url: http://www.code-industry.net/public/master-pdf-editor-4.3.89_qt5.amd64.tar.gz
```

I think that's the proper one. the URL is supposed to download the file.